test/action_policy/lookup_chain_test.rb

```diff
@@ -126,4 +134,16 @@ def test_lookup_namespace_by_policy_name
       ActionPolicy.lookup(a, namespace: LookupNamespace::NestedNamespace)
     )
   end
+
+  def test_lookup_chain_composition
+    assert_equal(
```

Comment:
Good catch - You're right. I'm not sure how to implement the skipping since `Rails` seems to be always defined at this point even when running in isolated mode.

I was wondering if we should get rid of the two tests for the chain composition because that's an implementation detail - what we actually care about is the lookup behaviour. What do you think?

Reply:
> if we should get rid of the two tests for the chain composition because that's an implementation detail

Agree 👍

Reply:
Ok great! That's updated now.